tools/code_format/check_format.py

```diff
@@ -159,6 +159,16 @@ def evaluate_lines(self, path, line_xform, write=True):
         format_flag = True
         output_lines = []
         for line_number, line in enumerate(self.read_lines(path)):
+            if line_number == 0 and path.endswith(".h"):
+                if line == "":
+                    error_message = "%s:%d: %s" % (
+                        path, line_number + 1, "the first line is empty for header files")
+                    continue
```

Comment:
```suggestion
            if line_number > 0 or not path.endswith(".h"):
                break
            if line == "":
                error_message = "%s:%d: %s" % (
                    path, line_number + 1, "the first line is empty for header files")
```

assuming the `line_numbers` are coming in order, we want to stop iterating as soon as they are `> 0`